docs/admin/authorization.md

```diff
@@ -85,6 +85,7 @@ Each line is a "policy object".  A policy object is a map with the following pro
       operations.
   - `resource`, type string; a resource from an URL, such as `pods`.
   - `namespace`, type string; a namespace string.
+  - `kubectl`, type boolean; when true, the user is allowed to access the /api endpoint (required for kubectl).
```

Comment:
We shouldn't start adding special cases to these attributes. I would be ok with a `nonResourceURL` or `nonResourcePath` attribute, so policy could address non-API things.